src/Mailer/Message.php

```diff
@@ -978,7 +978,7 @@ public function getHeadersString(array $include = [], string $eol = "\r\n", ?Clo
 
         $headers = [];
         foreach ($lines as $key => $value) {
-            if (empty($value) && $value !== '0') {
+            if (!$value && $value !== '0') {
```

Comment:
```suggestion
            if ($value === '') {
```

Reply:
Here theoretically no type is protecting the input. So this could change behavior if people entered some falsey values that are not empty string.